README.md

```diff
@@ -39,6 +39,7 @@ BD.Shopper.SellJunk() -- "Sells" all common junk items from players inventory by
 BD.Shopper.SellAlcohol() -- As above but for alcohol (consumables).
 BD.Shopper.SellJewellery() -- As above but for jewellery (pricier junk).
 BD.Shopper.SellAllUselessItems() -- As above but all at the same time.
+BD.Utility.Player.Respec() -- Respec attributes and perks
```

Comment:
Can this be moved to be under `BD.Cheats` since it's technically a cheat more than a utility?

Reply:
Done